### ISSUE TITLE
Occupancy mask

### ISSUE THREAD

**Issue by gottacatchenall** [EcoJulia/SimpleSDMLayers.jl#60]
expanding on #59 

one thing i found myself needing is wanting to create a SDMLayer that contained `1` at all locations with given occurrence data, and 0 elsewhere. the not optimized example i have is 
```
getOccupancyLayer(envLayer::SimpleSDMPredictor, occupancy) = begin
    occLayer = similar(envLayer)
    for o in occupancy
        long,lat = longitude(o), latitude(o) 
        occLayer[long,lat] = convert(eltype(occLayer), 1)
    end
    occLayer
end
```

**Comment by tpoisot** [comment]
wouldn't it be more julianesque to convert a `GBIFRecords` to a layer storing AbstractBool? I've been meaning to do this for a while now.

**Comment by tpoisot** [comment]
I settled on using `mask`, which can either do presence/absence or do a cumulative count of occurrences in a cell -- see #65

**Comment by gottacatchenall** [comment]
think this is closed as a product of #65
